docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3109,7 +3127,10 @@ One shots are programmed by choosing the `ActionProfileActionSet` message as the
 
 * `weight` specifying the probability of the action's selection at runtime. 0 is
   not a valid `weight` value and the server must return `INVALID_ARGUMENT` if
-  the client attempts to use it.
+  the client attempts to use it. The sum of all weights across all
+  `ActionProfileAction` messages for that `ActionProfileActionSet` message must
```

Comment:
Here is the "later" I referred to in an earlier comment, where it says sum of all weights, rather than number of entries.